src/Adapter/Module/Repository/ModuleRepository.php

```diff
@@ -29,6 +29,7 @@
 namespace PrestaShop\PrestaShop\Adapter\Module\Repository;
 
 use Exception;
+use Module;
```

Comment:
How can this be the only thing modified in this file ? didn't you forget to remove some `\`?

Reply:
Ah it's the remnant of a previous implementation I need to clean this indeed